core-web/libs/contenttype-fields/src/lib/fields/binary-field/binary-field.component.html

```diff
@@ -73,17 +73,21 @@
             data-testId="action-remove-btn"
             icon="pi pi-trash"></p-button>
     </div>
-
     <p-dialog
         [visible]="vm.dialogOpen"
         [modal]="true"
         [header]="dialogHeaderMap[vm.mode] | dm"
         [draggable]="false"
         [resizable]="false"
+        (onHide)="dialogClosed()"
         (visibleChange)="visibleChange($event)">
         <div [ngSwitch]="vm.mode">
             <div *ngSwitchCase="BINARY_FIELD_MODE.URL" data-testId="url">
-                TODO: Implement Import from URL
+                <dot-dot-binary-field-url-mode
+                    [accept]="accept"
+                    [maxFileSize]="maxFileSize"
+                    (tempFile)="setTempFile($event)"
+                    (cancel)="visibleChange(false)"></dot-dot-binary-field-url-mode>
```

Comment:
```suggestion
                    (cancel)="hideDialog(false)"></dot-dot-binary-field-url-mode>
```

or something like this, because this method only gets call here and only do the hiding.

Reply:
Do we need `visibleChange` and `dialogClosed` kinda looks like the same to me.